apps/web/src/components/v2Editor/customBlocks/python/PythonOutput.tsx

```diff
@@ -106,16 +107,33 @@ interface ItemProps {
   canFixWithAI: boolean
 }
 export function PythonOutput(props: ItemProps) {
+  const onExportToPNG = () => {
```

Comment:
There's a small issue here, which is that when you have more than one image output, clicking the `PNG` button only downloads the last one.

To reproduce, run this code and try downloading the PNG:

```python
import matplotlib.pyplot as plt
import numpy as np

# Sample data
x = np.linspace(0, 10, 100)
y = np.sin(x)
y2 = np.cos(x)

# Plotting sin(x) as the first output
plt.figure(figsize=(10, 6))
plt.plot(x, y, label='sin(x)')
plt.title('Plot of sin(x)')
plt.xlabel('x')
plt.ylabel('sin(x)')
plt.legend()
plt.grid(True)
plt.show()

# Plotting cos(x) as the second output
plt.figure(figsize=(10, 6))
plt.plot(x, y2, label='cos(x)', color='r')
plt.title('Plot of cos(x)')
plt.xlabel('x')
plt.ylabel('cos(x)')
plt.legend()
plt.grid(True)
plt.show()
```

By the way, this is not a big deal, I'm okay with merging with this issue and opening a separate issue to handle multi-output exports. I'll leave it up to you.

Reply:
Oh, didn't knew we could do that!

I'll try to handle multiple outputs in this PR.

Reply:
Fixed!